src/components/createbook.css

```diff
@@ -0,0 +1,55 @@
+.form {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+  padding-left: 1rem;
+  padding-right: 1rem;
```

Comment:
- [ ] Same comment as above, I suggest you to use the shorthand version to set your padding 🌿 

`padding: 2rem 1rem;`